src/jobflow_remote/jobs/daemon.py

```diff
@@ -355,6 +519,29 @@ def start_supervisord(
         nodaemon: bool = False,
         connect_interactive: bool = False,
     ) -> str | None:
+        """
+        Start the supervisord daemon.
```

Comment:
Correct me if I'm wrong. This starts the supervisord process + the processes right ? I got confused a bit about this in the start method below when the status of the daemon is SHUTDOWN, it does self.start_supervisord but it does not do start_processes afterwards, so I guess start_supervisord starts supervisord and it's processes from the config.

Reply:
In that case, maybe add a Note to explain that the processes are started (and that there is no need to do self.start_processes). It was not completely clear to me but maybe it's because I'm not fully aware of how supervisord works exactly.

Reply:
done